index.html

```diff
@@ -63,6 +93,76 @@ <h2 class="text-primary">Add a new Book</h2>
         </div>
       </form>
     </div>
+
+    <div class="my-4 max-width-800 visually-hidden" id="contact">
+      <h2 class="text-primary text-center">Contact Information</h2>
+      <p class="text-center">
+        Do you have any questions or you just want to say "Hello"?<br />
+        You can reach out to any of us!
+      </p>
+      <ul class="list-group">
+        <li
+          class="
+            list-group-item
+            d-flex
+            justify-content-between
+            align-items-center
+          "
+        >
+          KARANGWA NTWALI JOSUE
+          <span class="badge rounded-pill">
+            <a
+              class="
+                btn
+                list-group-item list-group-item-action list-group-item-primary
+              "
+              href="https://github.com/Ntwali-Josue"
+              >GITHUB</a
+            >
+          </span>
+        </li>
+        <li
+          class="
+            list-group-item
+            d-flex
+            justify-content-between
+            align-items-center
+          "
+        >
+          DANIEL SAMUEL
+          <span class="badge rounded-pill"
+            ><a
+              class="
+                btn
+                list-group-item list-group-item-action list-group-item-primary
+              "
+              href="https://github.com/DanSam5k"
+              >GITHUB</a
+            >
+          </span>
+        </li>
+        <li
+          class="
+            list-group-item
+            d-flex
+            justify-content-between
+            align-items-center
+          "
+        >
+          ZAIN SADAQAT
+          <span class="badge rounded-pill">
+            <a
+              class="
+                btn
+                list-group-item list-group-item-action list-group-item-primary
+              "
+              href="https://github.com/zainsadaqat"
+              >GITHUB</a
+            >
+          </span>
+        </li>
+      </ul>
```

Comment:
-  [x] The contents of your contact page does match the provided wireframe design template. As we want our design to as much as possible closely follow the the provided [wireframe template](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md) kindly ensure to:
- **remove borders** around the `Our email`, ` Our phone number` and `Our address` elements
- Provide a **realistic**  _email_, _contact_ and _address_ for `Our email`, ` Our phone number` and `Our address`
- align `Our email`, ` Our phone number` and `Our address` to the **left**, and **bulleted** as can be observed in the template design

**Your design:**
!![image](https://user-images.githubusercontent.com/12745474/129295859-0d6df213-3ab3-4fe9-801f-319cef225971.png)

**The wireframe design:**
![image](https://user-images.githubusercontent.com/12745474/129186849-0a49d1f3-e0c0-40aa-8528-0e755a14e25f.png)